### PR TITLE
Adding letter spacing

### PR DESCRIPTION
**Pull request by brianrclay**
@phantomxc @jgerlach, adding letter spacing so we can have better uppercase font styles.

### CODE REVIEW

**Comment by phantomxc** [comment]
I think we need the JSON and the Docs generated.

**Comment by brianrclay** [comment]
@phantomxc , done... I think

File: package.json [hunk 1]
```diff
@@ -30,4 +30,4 @@
     "@babel/preset-env": "^7.7.1",
     "babel-plugin-module-resolver": "^3.2.0"
   }
-}
\ No newline at end of file
+}
```
Comment (phantomxc) [inline package.json]: You can update the version in this file since we don't use lerna in this repo.

File: src/tokens/letterSpacing.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const light = core => ({
+  SmallUppercase: core.LetterSpacing.Uppercase,
```
Comment (jgerlach) [inline src/tokens/letterSpacing.js:2]: Are we planning on having different values for these in the future?

File: package-lock.json [hunk 3]
```diff
@@ -1,10 +1,6 @@
 {
   "name": "mx-design-tokens",
-<<<<<<< HEAD
-  "version": "8.1.1",
-=======
-  "version": "8.2.0",
->>>>>>> master
+  "version": "8.3.0",
```
Comment (phantomxc) [inline package-lock.json]: Delete this file, update the package.json as explained below and then run npm install.

File: src/tokens/letterSpacing.js (hunk 2 above)
Comment (brianrclay) [reply src/tokens/letterSpacing.js:2]: Possibly

**Comment by brianrclay** [comment]
Updated version, removed package-lock.json, and ran `npm i`

@phantomxc

File: src/tokens/letterSpacing.js (hunk 2 above)
Comment (jgerlach) [reply src/tokens/letterSpacing.js:2]: Do we need more than one then?

Comment (brianrclay) [reply src/tokens/letterSpacing.js:2]: Well we have 3 different uppercase styles (Tiny, XSmall, and Small). 

We might adjust values in the future. I'm not really sure.

**Pull request merged**